server/src/Server.hs

```diff
@@ -5,6 +5,9 @@ module Server
 )
 where
 
+import Data.Semigroup      ((<>))
```

Comment:
Please remove the superfluous spaces. We generally have our imports alphabetically sorted and without any alignment in order to prevent diffs consisting mostly of realignments.